static/app/components/events/autofix/autofixRootCause.tsx

```diff
@@ -224,41 +215,45 @@ function CauseOption({
             __html: singleLineRenderer(cause.title),
           }}
         />
-        <Button
-          icon={<IconChevron size="xs" direction={selected ? 'down' : 'right'} />}
-          aria-label={t('Select root cause')}
-          aria-expanded={selected}
-          size="zero"
-          borderless
-        />
+        <div style={{display: 'flex', flexDirection: 'row'}}>
```

Comment:
Could you apply these styles with a styled component instead? The `style` attribute cannot be overridden so we generally avoid using it for static styles.

Reply:
Done!